### PR TITLE
[RASDLG] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to PR https://github.com/reactos/reactos/pull/5908

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian for review

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 1]
```diff
@@ -2224,7 +2226,7 @@ BEGIN
     1580 "Această conexiune a fost configurată cu un nivel de securitate nesusținut de infrastructura programelor instalate.  Opțiunile de securitate ale acestei conexiuni au fost ajustate la nivelul de securitate disponibil.\0"
     1581 "Tip&ul de VPN:\0"
     1582 "Introduceți un nume de utilizator.\0"
-    1583 "Aceste date de autentificare sunt păstrate pentru propria dumneavoastră utilizare.  Totuși deja există un nume de utilizator disponibil pentru toți utilizatorii acestei conexiuni.  Doriți înlăturarea datelor de autentificare disponibile pentru toți ceilalți utilizatori?\0"
+    1583 "Aceste date de autentificare sunt salvate pentru propria dumneavoastră utilizare.  Totuși deja există un nume de utilizator disponibil pentru toți utilizatorii acestei conexiuni.  Doriți înlăturarea datelor de autentificare disponibile pentru toți ceilalți utilizatori?\0"
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:2229]:
```suggestion
    1583 "Aceste date de autentificare sunt salvate pentru propria dumneavoastră utilizare. Totuși deja există un nume de utilizator disponibil pentru toți utilizatorii acestei conexiuni. Doriți înlăturarea datelor de autentificare disponibile pentru toți ceilalți utilizatori?\0"
```

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 2]
```diff
@@ -1,7 +1,9 @@
 /*
- * FILE:       dll/win32/rasdlg/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * PROJECT:     ReactOS RASDLG
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:4]: added PURPOSE line useless in all your PRs.

Comment (AndreiMiloiu) [reply dll/win32/rasdlg/lang/ro-RO.rc]: Ok, I did it!

Comment (binarymaster) [reply dll/win32/rasdlg/lang/ro-RO.rc]: Again, this was wrong suggestion. Please revert back.

Comment (AndreiMiloiu) [reply dll/win32/rasdlg/lang/ro-RO.rc]: I understand, I will.

**Pull request merged**